GSG-G8/art-gallery
# 10 signup

### make register route for artist and customer
- validation inputs 
- check email if exist 
- create new user
- create cookie (after register user redirect to landing page )

![Screenshot from 2020-06-20 16-47-37](https://user-images.githubusercontent.com/47949938/85203348-d22bea80-b315-11ea-8a6c-0a39506ba1d7.png)
